source/lac/trilinos_sparse_matrix.cc

```diff
@@ -2295,7 +2295,9 @@ namespace TrilinosWrappers
       // matrix that we got as a result.
       Epetra_CrsMatrix *C_mat;
       ML_Operator2EpetraCrsMatrix(C_, C_mat);
-      C_mat->FillComplete();
+      C_mat->FillComplete(mod_B->DomainMap(),
+                          transpose_left ? inputleft.trilinos_matrix().DomainMap() :
+                          inputleft.trilinos_matrix().RangeMap());
```

Comment:
Nice! This is really all? ;-)

Please break the `?` case onto a line of its own to make it symmetric with the `:` case.